parsers/IN_WE.py

```diff
@@ -210,7 +209,7 @@ def fetch_consumption(
     logger: Logger = getLogger(__name__),
 ) -> list[dict[str, Any]]:
     if target_datetime is None:
-        target_datetime = arrow.utcnow().datetime
+        target_datetime = datetime.now(TZ)
```

Comment:
Changing the timezone of this from UTC also changes the returned datetimes, I'm not sure what is correct but let's put it back to UTC for now (and in the fetch exchange function).

@mathilde-daugy do you know if this has to be in UTC or not?

Reply:
the datetimes should be localized either way, as long as it is consistent across the script then there shouldn't be an error

Reply:
Okay, I'll create a suggestion to fix this then :)